css/acquia_lift.personalize.theme.css

```diff
@@ -1,9 +1,9 @@
-/* Page variation edit toggle in unified navigation bar */
```

Comment:
The toggle is removed so this whole section that was affected here can be removed.